repo/packages/D/drone-agent/0/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "packagingVersion": "3.0",
+    "description": "Drone is a Continuous Delivery platform built on Docker, written in Go.  This is the drone-agent; you MUST install the drone-server first.",
+    "framework": false,
+    "licenses": [
+        {
+            "name": "Apache license",
+            "url": "https://raw.githubusercontent.com/drone/drone/master/LICENSE"
+        }
+    ],
+    "maintainer": "brad@drone.io",
```

Comment:
Might be better to find someone else who is actually willing and able to maintain this package. Brad is spread incredibly thin.